<!--
-->
<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video1" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="/camera_link" />
    <param name="framerate" value="30" />
    <param name="contrast" value="32" />
    <param name="brightness" value="32" />
    <param name="saturation" value="32" />
    <param name="autofocus" value="false" />
    <param name="focus" value="51" />
    <!-- <param name="do_rectify" value="true" /> not working --> 
    <param name="camera_info_url" value="package://line_detection/config/head_camera.yaml" /> 
    <param name="camera_name" value="head_camera" />
    <param name="io_method" value="mmap"/>
  </node>
    <!--
    <node name="img_proc" pkg="image_proc" type="image_proc" ns="usb_cam">

        <param name="queue_size" value="5" />
    </node>
    -->
    <node name="line_detector" pkg="line_detection" type="line_detection_node" output="screen" > 
    </node>

  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/perception/floor_lines"/>
    <param name="autosize" value="true" />
  </node>
</launch>
